drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -412,12 +382,120 @@ def default_background_image_url(self):
         """ The URL to the default background image, shown when no custom background is used """
         return self.runtime.local_resource_url(self, "public/img/triangle.png")
 
+    @property
+    def attemps_remain(self):
+        return self.max_attempts is None or self.num_attempts < self.max_attempts
```

Comment:
We might want to allow both `None` (empty) and `0` values of `max_attempts` to mean "unlimited attempts". It would be consistent with some other problem types where `0` means no limit.